solr/datadog_checks/solr/data/metrics.yaml

```diff
@@ -126,11 +126,29 @@ jmx_metrics:
       category: QUERY
       name: requestTimes
       attribute:
-        avgRequestsPerSecond:
+        MeanRate:
           alias: solr.search_handler.avg_requests_per_sec
           metric_type: gauge
-        avgTimePerRequest:
-          alias: solr.search_handler.avg_time_per_req
```

Comment:
note: `avgTimePerRequest` does not exist anymore since 7.0. https://lucene.apache.org/solr/guide/8_5/performance-statistics-reference.html#commonly-used-stats-for-request-handlers

Reply:
Do we need to keep submitting these for users monitoring versions < 7.0?

Reply:
@ChristineTChen Yes, it's done here: https://github.com/DataDog/integrations-core/blob/789cba4ba64973062481abf24d257b10d78e68e8/solr/datadog_checks/solr/data/metrics.yaml#L216-L221